mmseg/models/segmentors/base.py

```diff
@@ -184,7 +190,10 @@ def postprocess_result(self,
                 'seg_logits':
                 PixelData(**{'data': i_seg_logits}),
                 'pred_sem_seg':
-                PixelData(**{'data': i_seg_pred})
+                PixelData(**{'data': i_seg_pred}),
+                'gt_sem_seg':
+                PixelData() if only_prediction else PixelData(
```

Comment:
Is it necessary to add an empty PixelData for `gt_sem_seg` when only predict?

Reply:
removed